aat/auth_test.go

```diff
@@ -67,45 +60,34 @@ func TestJoinRealmWithCRCookieAuth(t *testing.T) {
 
 	if scheme == "ws" || scheme == "wss" {
 		jar, err := cookiejar.New(nil)
-		if err != nil {
-			t.Fatal("failed to create CookieJar:", err)
-		}
+		require.NoError(t, err, "failed to create CookieJar")
```

Comment:
btw, authors suggest that if you assert many times, use the below:
```go
assert := assert.New(t)
// or in our case
assertions := require.New(t)
```

So we can skip `t` first param in every assertion. and also it seems that this should be faster...

Reply:
> authors suggest that if you assert...
As far as using `require.New(t)`, I opted not to do that as all it does is call the corresponding `require.func`. [Example](https://github.com/stretchr/testify/blob/v1.8.4/require/require_forward.go#L132). That ends calling the helper type assertion twice. ...not that I care about performance in test code. Getting rid of that first param did not seem that important, but if you have any strong preference I will be happy to.

Reply:
Nope It's ok! I think this is a matter of preferences only.